### PR TITLE
USER frame with test

### PR DESCRIPTION
**Pull request by m13m**

### CODE REVIEW

File: test/id3vx/encoding_test.exs [hunk 1]
```diff
@@ -349,4 +349,26 @@ defmodule Id3vx.EncodingTest do
...
+
+    binary = Frame.encode_frame(frame, %Tag{version: 3})
+    assert <<frame_header::binary-size(10), frame_data::binary>> = binary
+    assert <<"USER", frame_size::size(32), _flags::binary-size(2)>> = frame_header
+    assert 20 == frame_size
+    assert <<0x01::size(8), frame_rest::binary>> = frame_data
+    <<language::binary-size(3), text::binary>> = frame_rest
+
+    assert language == frame.data.language
+    assert text == frame.data.text
```
[the remark this answers is not in the document]
Comment (m13m) [reply test/id3vx/encoding_test.exs:394]: this is how `convert_string` does the conversion for `utf16` 

```

  defp convert_string(:utf16, data) do
    {encoding, bom_length} = :unicode.bom_to_encoding(data)
    {_, string_data} = String.split_at(data, bom_length)
    :unicode.characters_to_binary(string_data, encoding)
  end

```

[the remark this answers is not in the document]
Comment (m13m) [reply test/id3vx/encoding_test.exs:394]: I wonder should I rewrite the the convert_string function with this `"data" |> :unicode.characters_to_binary(:utf8, :utf16)"`

[the remark this answers is not in the document]
Comment (lawik) [reply test/id3vx/encoding_test.exs:394]: If you think that's the problem. You can try it out.

**Comment by lawik** [comment]
This has conflicts and I don't know what the recent updates are :) If you can summarize I know what to look at.

**Comment by m13m** [comment]
@lawik this PR is ready for review and added test where directly check with id3v2 cli to form the existence of USER frame in the binary.

**Comment by m13m** [comment]
- Fixed conflicts 
- it checks the presence USER frame with ffmpeg and id3v2 cli tool.

**Pull request merged**